frontend/src/app/main/data/tokens.cljs

```diff
@@ -224,6 +231,7 @@
           (rx/of
            (drop-error (ex-data e))))))))
 
+;; FIXME: the the name is very confusing
```

Comment:
I don't think the name is confusing... `update-create` is a common pattern for udating an object if it exists, or creating it if not...

Reply:
The name is clearly confusing, the common pattern is `create-or-update-<subject>`.
In any case this is already refactored and changed  on part3 PR